os/include/tinyara/serial/tioctl.h

```diff
@@ -202,6 +202,7 @@
 #define TIOCCLRSIG      _TIOC(0x002f)	/* Clear signal to handle Interrupt */
 #define TIOCTXSTATE     _TIOC(0x0030)	/* Get State of TX */
 #define TIOCRXSTATE     _TIOC(0x0031)	/* Get State of RX */
+#define TIOCLOOPBACK    _TIOC(0x0032)	/* set loopback mode */
```

Comment:
comment was not appropriate for this cmd. corrected now.
we can not rename it like `TIOCSETLO`  because we doing both operations with this cmd, enable and disable.